int8 total
uint64 timestamp
float64[32] x
float64[32] y
int8[32] player_id
string[32] tag_id
float64[32] size
